packages/augur-state/src/run.worker.ts

```diff
@@ -0,0 +1,19 @@
+// have to use require in workers so webpack is happy
+const Run = require("./run");
+
+const ctx: Worker = self as any;
+
+ctx.addEventListener('message', event => {
+  // XXX: todo, add routing code here
+  ctx.postMessage("Response boo");
+});
+
+// send everything to the browser
+console.log = (msg: string) => {
+  ctx.postMessage(msg);
+}
+
+Run.start();
+
+// to stop typescript from complaining
```

Comment:
Just curious: what is typescript saying?

Reply:
ERROR in /Users/brian/work/augur/packages/augur-state/src/index.ts
./src/index.ts
[tsl] ERROR in /Users/brian/work/augur/packages/augur-state/src/index.ts(5,23)
      TS2306: File '/Users/brian/work/augur/packages/augur-state/src/run.worker.ts' is not a module.

Reply:
Ah. From this and how you had to define `ctx`, I get the impression that typescript is still a little unprepared for web workers.